Doc/Manual/Go.html

```diff
@@ -60,38 +60,44 @@
 
 
 <p>
-Go is a compiled language, not a scripting language.  However, it does
-not support direct calling of functions written in C/C++.  The cgo
-program may be used to generate wrappers to call C code from Go, but
-there is no convenient way to call C++ code.  SWIG fills this gap.
+Go does not support direct calling of functions written in C/C++.  The
+<a href="https://golang.org/cmd/cgo/">cgo program</a> may be used to generate
+wrappers to call C code from Go, but there is no convenient way to call C++
+code.  SWIG fills this gap.
 </p>
 
 <p>
-There are (at least) two different Go compilers.  One is the gc
-compiler, normally invoked via the go tool.  The other
-is the gccgo compiler, which is a frontend to the gcc compiler suite.
-The interface to C/C++ code is completely different for the two Go
-compilers.  SWIG supports both, selected by a command line option.
+There are (at least) two different Go compilers.  The first is the Go compiler
+of the <a href="https://golang.org/doc/install">Go distribution</a>.
+Since Go 1.5 the Go compiler is part of the <a href="https://golang.org/cmd/go/">
```

Comment:
These lines aren't accurate.  The only change in 1.5 is a change in the name of the compiler, which is an unimportant detail for most people.  I think the existing sentence is accurate: "One is the gc compiler, normally invoked via the go tool."  You can rework that to refer to the Go distribution, but there shouldn't be any notes about the compiler changing in 1.5.